### ISSUE TITLE
Exception when execute ScrollAll under .Net Core (works under .Net Framework)

### ISSUE THREAD

**Issue by Gimli62** [elastic/elasticsearch-net#4051]
**NEST/Elasticsearch.Net version**:  7.2.1

**Elasticsearch version**: 7

**Description of the problem including expected versus actual behavior**:
I have code performing scrolling query  that works fine under .Net Framework 7.4.2 but when rebuild under .Net Core and executed in that environment raise an Elastic exception as shown below.
Expected results is to receive documents so that they can be added into list. 

I use NEST 7.2.1, .Net standard library 2.0.3

**Steps to reproduce**:
 1.
 2.
 3.

**Provide `ConnectionSettings` (if relevant)**:

**Provide `DebugInformation` (if relevant)**:

**Exception stack **
Elasticsearch.Net.ElasticsearchClientException
HResult=0x80131500
Message=scrolling search on /coint%2A/_search?typed_keys=true&scroll=5m with slice 0 was not valid on scroll iteration 0
Source=Nest
StackTrace:
at Nest.BlockingSubscribeExtensions.WaitOnObservable[TObservable,TObserve,TObserver](TObservable observable, TimeSpan maximumRunTime, Func3 factory) at Nest.BlockingSubscribeExtensions.Wait[T](IObservable1 observable, TimeSpan maximumRunTime, Action`1 onNext)
at Slb.Prism.DevTool.Library.RollupFlowCalls.Search.getList(ElasticClient esClient, String index, ILogger log) in D:<blabla>\DevTool.Service.ControlFlow\RollupFlowCalls\Search.cs:line 53
at ConsoleTestCore.Program.Main(String args) in D:<blabla>\ControlFlow\ConsoleTestCore\Program.cs:line 15

** Code to reproduce issue ***
```
public static List<Record> getList(ElasticClient esClient, string index, ILogger log)
    {
        if (log != null) log.LogInformation($"Entersearch");
        var seenDocuments = 0;
        var doccuments = new ConcurrentBag<IReadOnlyCollection<Source>>();
        DateTime gteTime = DateTime.Now.AddMinutes(-10);
        DateTime ltTime = DateTime.Now;
        List<Record> indexedList = new List<Record>();
        int para = 6;

        if (log != null) log.LogInformation($"Start search");
        esClient.ScrollAll<Source>("5m", para, s => s
        .MaxDegreeOfParallelism(para )
        .Search(w => w
        .Index(index)
                        .Source(true)
                        .From(0)
                        .Size(500)
                        .Query(q => q
                            .Bool(b => b
                                .Filter(bf => bf
                                    .DateRange(dr => dr
                                       .Field(f => f.timestamp)
                                       .GreaterThanOrEquals(gteTime)
                                       .LessThan(ltTime)))
                                .Must(mu => mu
                                    .Match(ma => ma
                                        .Field(fi => fi.level)
                                        .Query("Stack")))))))
        .Wait(TimeSpan.FromMinutes(5), r =>
        {
            if (log != null) log.LogInformation($"In wait");
            doccuments.Add(r.SearchResponse.Documents);
        });
        if (log != null) log.LogInformation($"Out wait");

     return null ; // just to be simple
}
```

**Comment by Mpdreamz** [comment]
Are you able to capture the `Elasticsearch.Net.ElasticsearchClientException` and print what its `DebugInformation` property says? That should give us some insight.

**Comment by Gimli62** [comment]
Yes, here is the content :

'($exception).DebugInformation' threw an exception of type 'System.NullReferenceException'

![elastic exception](https://user-images.githubusercontent.com/12194392/64244343-71c0ed80-cf3b-11e9-859e-421fdcd35c8d.png)

**Comment by Gimli62** [comment]
Any update ?

**Comment by russcam** [comment]
@Gimli62 this may possibly be related to https://github.com/elastic/elasticsearch-net/pull/4077. This fix will be in the next release

**Comment by russcam** [comment]
@Gimli62 would you be able to try this again with the latest NEST 7.x version, now that #4077 is merged and in the released versions from 7.3.1 onwards?

**Comment by codebrain** [comment]
Hi @Gimli62, I am closing this ticket as we were awaiting a response with the debug information, utilizing the new client. You are welcome to resubmit a new issue if this is still causing you problems.
